build.gradle

```diff
@@ -112,8 +80,8 @@ project.tasks['sonarqube'].dependsOn jacocoTestReport
 
 sonarqube {
   properties {
-    property "sonar.projectName", "Reform :: spring-boot-template"
-    property "sonar.projectKey", "uk.gov.hmcts.reform:spring-boot-template"
+    property "sonar.projectName", "Reform :: bulk-scan-end-to-end-tests"
```

Comment:
do we need sonar ?

Reply:
if sonar lints test code, then I think it's worth leaving it in.